### PR TITLE
Icon (symbolics) theming

### PR DESCRIPTION
**Pull request by gweebg**
### Description
This pull request continues the idea of https://github.com/corecoding/Vitals/pull/295 and aims to implement the feature exposed on https://github.com/corecoding/Vitals/issues/295#issuecomment-1460100965 by @corecoding by adding an icon theme switcher in the preference's menu.

### Changes to the UI

The icon theme switcher is made out of a combo box on the general section of the preference's menu, and, as it is, includes two themes to choose from, `original` and `gnome`. The icons from `original` are the current theme and the icons from `gnome` are the icons proposed by @daudix in https://github.com/corecoding/Vitals/pull/295.

Interface updates

Some changes to the layout had to be made, in specific, the section "About", with the addition of the new combo box would lose alignment with the section "Sensors", so I took the freedom to move it to the bottom. Another improvement to the UI is the definition of `margin-end` to all switch components, so that they wouldn't be touching the `GtkBox` right limit.

**Note**: Currently, the `gnome` theme includes the same `gpu` symbolic icon as the `original` theme. If this PR gets merged, I would ask @daudix for an updated version since he is the one responsible for the theme (🥺).

### Changes to `/icons`

In order to have multiple themes, each theme is now represented as a new folder on the `Vitals/icons` directory and the name of the icons SVG included on a theme must match the names of the `original` theme. In this pull request we have the `/original` and `/gnome` folders, each representing a theme.

#### Adding a new theme

To add a new icon theme:
1. Create the folder at `/icons` with the new symbolics;
2. Update the combo box items in `prefs.ui`;
3. Append the new theme path to the `_sensorsIconPathPrefix` variable in `extension.js`;

Note that on step `3.`, the index of each theme's prefix on the `_sensorsIconPathPrefix` variable must match the index of the combo box item defined at `prefs.ui`. For example, if on the combo box the `original` theme is the first item (index 0) then it must be the first on the list of prefixes.

### README Updates

Updated the README.md file to include the credits for the newly added theme.

### CODE REVIEW

**Comment by daudix** [comment]
I would suggest naming the new theme "Adwaita" or "GNOME", since later on other icon themes could be added

**Comment by gweebg** [comment]
> I would suggest naming the new theme "Adwaita" or "GNOME", since later on other icon themes could be added

Done! Changed the new theme name from `updated` to `gnome`, and updated the README.

File: README.md [hunk 1]
```diff
@@ -86,14 +86,22 @@ Vitals is a GNOME Shell extension for displaying your computer's temperature, vo
...
+
+### Original Theme
 * (voltage|fan)-symbolic.svg - inherited from Freon project.
 * (system|storage)-symbolic.svg - from Pop! OS theme.
 * temperature-symbolic.svg - [iconnice studio](https://www.iconfinder.com/iconnice).
 * (cpu|memory)-symbolic.svg - [DinosoftLabs](https://www.iconfinder.com/dinosoftlabs).
 * network\*.svg - [Yannick Lung](https://www.iconfinder.com/yanlu).
 * Health icon - [Dod Cosmin](https://www.iconfinder.com/icons/458267/cross_doctor_drug_health_healthcare_hospital_icon).
 
+### Gnome Theme
```
Comment (daudix) [inline README.md:99]: GNOME, not Gnome ;)

Comment (gweebg) [reply README.md]: Fixed, on both README and UI combo box!

**Review by corecoding — APPROVED** [review]

**Comment by corecoding** [comment]
Thank you, this is great!

**Pull request merged**